soc/riscv/riscv-privilege/telink_b9x/soc.c

```diff
@@ -109,6 +116,41 @@ void soc_load_rf_parameters_deep_retention(void)
 }
 #endif
 
+bool flash_cap_check(void) {
```

Comment:
Rename to more clean way like flash_cap_suitable. In that way it's clean true/false meaning.

Reply:
fixed